File: backend/Dtos/LoanSnapshotPhotoDto.cs
namespace backend.Dtos
{

    public class LoanSnapshotPhotoDto
    {
        public int Id { get; set; }
        public int LoanId { get; set; }
        public string PhotoUrl { get; set; } = string.Empty;
        public int DisplayOrder { get; set; }
        public DateTime SnapshotTakenAt { get; set; }
    }


    public class LoanSnapshotPhotoListDto
    {
        public int Id { get; set; }
        public string PhotoUrl { get; set; } = string.Empty;
        public int DisplayOrder { get; set; }
    }

    
}

File: frontend/src/app/dtos/loanSnapshotPhotoDto.ts
export interface LoanSnapshotPhotoDto {
  id: number;
  loanId: number;
  photoUrl: string;
  displayOrder: number;
  snapshotTakenAt: string;
}

export interface LoanSnapshotPhotoListDto {
  id: number;
  photoUrl: string;
  displayOrder: number;
}